src/Uno.UI/UI/Xaml/UIElement.Keyboard.skia.cs

```diff
@@ -45,13 +67,6 @@ private void CoreWindow_KeyDown(CoreWindow sender, KeyEventArgs args)
 						$"ScanCode: {args.KeyStatus.ScanCode})"
 					);
 				}
-
-				var originalSource = FocusManager.GetFocusedElement() as UIElement ?? Window.Current.Content;
-
-				originalSource.RaiseEvent(
-					KeyDownEvent,
-					new KeyRoutedEventArgs(originalSource, args.VirtualKey, args.KeyStatus) { CanBubbleNatively = false }
-				);
```

Comment:
Hmmm.. looks like this breaks catalyst?

The following line will effectively do "nothing"

https://github.com/unoplatform/uno/blob/6f1d732fd0fa79f6994b01694e9f86e75c7ccd39/src/Uno.UI/Controls/Window.iOS.cs#L106

Reply:
My bad. I thought this is a shared code.